src/objects.js

```diff
@@ -1,26 +1,29 @@
 const createPerson = (name, age) => {
-  // your code here
+    return person = {
+      name: name,
+      age: age
+    }
```

Comment:
Again, return values, not assignment:

```js
return {
  name: name,
  age: age
}
```

One cool thing you can do with objects in ES6, is if your key and variable share the same name, you can omit the colon and the variable.

```js
return {
  name,
  age
}
```

See [Object Literal Property Value Shorthand](https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand).